onboard/catkin_ws/src/acoustics/scripts/acoustics_wrapper.py

```diff
@@ -24,10 +26,12 @@ def __init__(self):
         self.client_sampling = actionlib.SimpleActionClient('call_saleae', SaleaeAction)
```

Comment:
For this, you need to make sure that you only start either the saleae client or the generate_data client, not both. Read in a ros parameter that determines which you are using, and only start the one.